wpilibcExamples/src/main/cpp/templates/commandbased/Robot.cpp

```diff
@@ -5,93 +5,60 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include <Commands/Command.h>
+#include "Robot.h"
+
 #include <Commands/Scheduler.h>
-#include <LiveWindow/LiveWindow.h>
-#include <SmartDashboard/SendableChooser.h>
 #include <SmartDashboard/SmartDashboard.h>
-#include <TimedRobot.h>
 
-#include "Commands/ExampleCommand.h"
-#include "Commands/MyAutoCommand.h"
+ExampleSubsystem Robot::subsystem;
 
-class Robot : public frc::TimedRobot {
-public:
-	void RobotInit() override {
-		m_chooser.AddDefault("Default Auto", &m_defaultAuto);
-		m_chooser.AddObject("My Auto", &m_myAuto);
-		frc::SmartDashboard::PutData("Auto Modes", &m_chooser);
-	}
+void Robot::RobotInit() {
+	m_chooser.AddDefault("Default Auto", &m_defaultAuto);
+	m_chooser.AddObject("My Auto", &m_myAuto);
+	frc::SmartDashboard::PutData("Auto Modes", &m_chooser);
+}
 
-	/**
-	 * This function is called once each time the robot enters Disabled
-	 * mode.
-	 * You can use it to reset any subsystem information you want to clear
-	 * when
-	 * the robot is disabled.
-	 */
-	void DisabledInit() override {}
+void Robot::DisabledInit() {}
 
-	void DisabledPeriodic() override {
-		frc::Scheduler::GetInstance()->Run();
-	}
+void Robot::DisabledPeriodic() {
+	frc::Scheduler::GetInstance()->Run();
+}
 
-	/**
-	 * This autonomous (along with the chooser code above) shows how to
-	 * select
-	 * between different autonomous modes using the dashboard. The sendable
-	 * chooser code works with the Java SmartDashboard. If you prefer the
-	 * LabVIEW Dashboard, remove all of the chooser code and uncomment the
-	 * GetString code to get the auto name from the text box below the Gyro.
-	 *
-	 * You can add additional auto modes by adding additional commands to
-	 * the
-	 * chooser code above (like the commented example) or additional
-	 * comparisons
-	 * to the if-else structure below with additional strings & commands.
-	 */
-	void AutonomousInit() override {
-		std::string autoSelected = frc::SmartDashboard::GetString(
-				"Auto Selector", "Default");
-		if (autoSelected == "My Auto") {
-			m_autonomousCommand = &m_myAuto;
-		} else {
-			m_autonomousCommand = &m_defaultAuto;
-		}
+void Robot::AutonomousInit() {
+	std::string autoSelected = frc::SmartDashboard::GetString(
```

Comment:
Line 28-34 should be commented out (in accordance with the comment for the function)